Makefile

```diff
@@ -121,7 +121,7 @@ test-integration: manifests fmt vet envtest ginkgo ## Run integration tests.
 	$(GINKGO) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 
 .PHONY: test-e2e
-test-e2e: manifests generate fmt vet envtest ginkgo kind-image-build
+test-e2e: manifests generate fmt vet envtest ginkgo kustomize kind-image-build
```

Comment:
1. remove "generate", it is not needed
2. In e2e-test.sh, line 22, use "kubectl" instead of "$KUBECTL"